### DISCUSSION TITLE
Enhancement: not existing Exception in documentation

### DISCUSSION THREAD

**Discussion by JonasRueegsegger** [miguelgrinberg/Flask-SocketIO#2030] [General]
In the connect section of the getting started documentation https://flask-socketio.readthedocs.io/en/latest/getting_started.html#connection-events you are mentioning an exception which does not exist in the latest package version:
from flask_socketio import ConnectionRefusedError
i guess you meant
from socketio.exceptions import ConnectionRefusedError ?

p.s. The package is really helpful :)

**Comment by miguelgrinberg** [comment]
I think the class exists. It's defined here: https://github.com/miguelgrinberg/Flask-SocketIO/blob/main/src/flask_socketio/__init__.py#L22
